README.md

```diff
@@ -1,7 +1,18 @@
 # About
 
+[![Build Status](https://travis-ci.org/openaire/vipe.png?branch=master)](https://travis-ci.org/openaire/vipe)
+
```

Comment:
Yeah. I have used it in [avroknife project](https://github.com/CeON/avroknife) before.